go.mod

```diff
@@ -1,6 +1,6 @@
 module helm.sh/helm/v3
 
-go 1.21
+go 1.22
```

Comment:
It appears the [codeql](https://codeql.github.com/) (security scanner) does not support Go 1.22 being declared in the `go.mod` file. The change has been made but not yet released.

Can you undo this change for now? We can make it later.

Reply:
Reverting this now.